tests/cache/tests.py

```diff
@@ -1834,6 +1834,18 @@ def test_redis_pool_options(self):
         self.assertEqual(pool.connection_kwargs["socket_timeout"], 0.1)
         self.assertIs(pool.connection_kwargs["retry_on_timeout"], True)
 
+    def test_delete_many_no_keys(self):
+        # Behaviour when no keys are passed should not raise
+        # redis.exceptions.ResponseError: wrong number of arguments
+        # for 'del' command
+        self.assertIsNone(cache.delete_many([]))
+
+    def test_set_many_empty_data(self):
+        # Behaviour when an empty dict is passed should not raise
+        # redis.exceptions.ResponseError: Command # 1 (MSET)
+        # of pipeline caused error: wrong number of arguments for 'mset' command
+        self.assertEqual(cache.set_many({}), [])
```

Comment:
I'd move these tests to the `BaseCacheTests` and remove comments.

Reply:
Ok.

Will check also if those 2 tests been placed to the basetest will introduce issues to the other backends and fix accordingly if needed

Reply:
I checked locally. It should pass for other backends.

Reply:
I made the changes locally. By moving the tests to the base class and also extending them to check for keys or data being None, 

e.g.

```
self.assertIsNone(cache.delete_many(None))
```

tests fail cause core/cache/backends/base.py

does not account for keys or data been None.

Tests either fail with

AttributeError: 'NoneType' object has no attribute 'items'

TypeError: 'NoneType' object is not iterable.

Are you interested in covering for such a case in the base class and therefore do you want me to extend the check for keys and data in BaseCache or is this something that is out of the scope of the framework design? (in the sense that the developer should account for those cases when calling delete_many or set_many)?